azure-pipelines.yml

```diff
@@ -11,6 +11,12 @@ jobs:
     pool:
       vmImage: 'ubuntu-latest'
```

Comment:
Do we need to move to ubuntu-20.04 now? Or can we simply set the `vmImage` to 18?

Reply:
That is the other fix.  I changed java & python ver. to keep this 'ubuntu-latest' to detect any other change later.
If it's unnecessary, it's ok to change it to 18

Reply:
Can we set this to ubuntu18 since internal build is (will be) on ubuntu18? You can leave other changes if they work fine on 18.

Reply:
Let's keep other changes for reference. Thanks!